pages/medizin/index.js

```diff
@@ -3,17 +3,81 @@ import { StyledHeading } from "../schlafen";
 import { ListItem } from "../../components/FacilityList";
 import Facility from "../../components/Facility";
 import { StyledList } from "../../components/Facility";
+import { useState } from "react";
 
-export default function Medizin({ facilities }) {
-  const filtered = facilities.filter((facility) => {
-    return facility.category.includes("Medizin");
+export default function Medicin({ facilities }) {
+  const medicinFacilitis = facilities.filter((facility) => {
+    return facility.category === "medicin";
   });
 
+  const [selectedFilters, setSelectedFilters] = useState([]);
+
+  function handleToggleFilter(event) {
+    const selectedValue = event.target.value;
+    console.log(selectedValue);
+    if (selectedFilters.includes(selectedValue)) {
+      const updatedFilters = selectedFilters.filter(
+        (selectedFilter) => selectedFilter !== selectedValue
+      );
+      setSelectedFilters(updatedFilters);
```

Comment:
ideally, since the new value is calculated based on a previous value, we use a callback here, similar to what you did in the `else` block.

```suggestion
      setSelectedFilters((previousSelectedFilters) => previousSelectedFilters.filter(
        (selectedFilter) => selectedFilter !== selectedValue
      ));
```